app/javascript/components/PostEditor.js

```diff
@@ -98,21 +113,20 @@ class PostEditor extends React.Component {
 					</p>
 				) : null}
 				<HtmlEditor
-					onChange={doc => this.handleFormChange(doc)}
+					onChange={this.handleFormChange}
 					onHasChanges={() =>
 						this.setState({ lastUnsavedChangeAt: new Date() })
 					}
-					value={postBody}
+					html={postBody}
+					pluginState={pluginState}
 					options={options}
 					autoFocus
 					render={({ editor, view }) => (
 						<div>
 							<Floater view={view}>
-								<MenuBar menu={{ marks: menu.marks }} view={view} />
+								<MenuBar menu={{ ...menu }} view={view} />
```

Comment:
this enables everything, change to something like `{{ comments: menu.comments, marks: menu.marks }}` to pick which sections to show